src/api/posts.ts

```diff
@@ -0,0 +1,25 @@
+import { Post } from '../react-app-env';
+
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
Same as in this comment https://github.com/Watcher1986/review-task3/pull/1/files#r1192135073.